src/Microsoft.AspNetCore.WebHooks.Receivers.AzureAlert/Extensions/AzureAlertMvcBuilderExtensions.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel;
+using Microsoft.AspNetCore.WebHooks;
+using Microsoft.AspNetCore.WebHooks.Internal;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Extension methods for setting up Azure Alert WebHooks in an <see cref="IMvcBuilder" />.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public static class AzureAlertMvcBuilderExtensions
```

Comment:
These additions are just boilerplate. And, I confirmed these files are almost identical to the corresponding `IMvcCoreBuilder` extension class, modulo `IMvcCoreBuilder` -> `IMvcBuilder` of course.

The meat is really in the `XyzServiceCollectionSetup` classes.

Reply:
OK cool beans